pkg/build/builder/docker_test.go

```diff
@@ -192,6 +192,14 @@ func TestDockerfilePath(t *testing.T) {
 		"\"io.openshift.build.commit.message\"=\"message\"",
 	}
 
+	if isNew, err := util.CreateBaseDir(util.GetBaseDir()); err == nil {
```

Comment:
thanks @smarterclayton, indeed, i should simply use` ioutil.TempDir("", "dockerfile-path")` @bparees sorry, how do you think, should i create a new pull request ?